project/src/Luna/Compilation.hs

```diff
@@ -71,7 +71,7 @@ type instance Inputs  Event ProjectCompilation = '[]
 type instance Outputs Event ProjectCompilation = '[New // AnyExpr]
 type instance Preserves     ProjectCompilation = '[]
 
-data ModuleCompilationError = ModuleSourcesNotFound QualName
+data ModuleCompilationError = ModuleSourcesNotFound [QualName] QualName
```

Comment:
Potentially stupid question. What is the additional `QualName` for? It's unclear from the code - could a type alias make it clearer?

Reply:
According to our "style guide" in single constructor data types we should always name fields unless its really completely obvious what it its. We should name them here.

Reply:
It's not a single constructor type though!

Reply:
Oh, then either we should use here type aliases or convert it to single field containing a single constructor data type. We do not create non-total functions unless absolutely necessary. So, as you suggested!